app/views/content/life-as-a-teacher/ages-and-specialisms/age-groups-you-could-teach.md

```diff
@@ -1,12 +1,8 @@
 ---
-title: Who do you want to teach
-heading: Who do you want to teach?
-subcategory: Who to teach
+title: Age groups you could teach
+heading: Age groups you could teach
 description: |-
   Explore the key stages and ages you want to teach, whether that's primary, secondary, early years, further education, or pupils with special educational needs.
```

Comment:
Wonder if this should be 'Explore the key stages and ages you could teach/might want to teach'?